<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrvManagerDialog</class>
 <widget class="QDialog" name="PrvManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="tree">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_copy_to_local_disk">
       <property name="text">
        <string>Copy to local disk</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_upload_to_server">
       <property name="text">
        <string>Upload to server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_done">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
